server/controllers/dashboardController.ts

```diff
@@ -1,10 +1,12 @@
 import { Request, RequestHandler, Response } from 'express'
+import paths from '../paths/api'
```

Comment:
I think we should create a new `paths/find.ts` file for this - we're using the API's `/programmes` path here which will change to `/courses` in the future, but we want to keep it as `/programmes` in the client.